### ISSUE TITLE
add index to provided slot scope

### ISSUE THREAD

**Issue by mmouterde** [albertodeago/vue-virtualized-list#12]
I would be great to get the index of the item in the whole list :
```html
<virtualized-list [...]>
                        <template v-slot="{item, **index**}">
                        </template>
</virtualized-list>
```

**Comment by mmouterde** [comment]
Please consider PR https://github.com/albertodeago/vue-virtualized-list/pull/13

**Comment by albertodeago** [comment]
Hi @mmouterde, sorry for the delay, I'm going to look at this today, many thanks for the PR submission!

**Comment by albertodeago** [comment]
Closed with PR #13
